src/js/utils/ValidatorUtil.js

```diff
@@ -34,22 +34,22 @@ var ValidatorUtil = {
   },
 
   isInteger: function (value) {
-    const number = parseFloat(value);
-
-    return Number.isInteger(number);
+    return ValidatorUtil.isNumber(value) &&
+      Number.isInteger(parseFloat(value));
```

Comment:
Is doing `parseFloat` a good idea?
Number.isInteger(parseFloat('1.1')); // true
Number.isInteger('1.1'); // false
If the calling function doesn't end up doing `parseFloat` it may have unexpected results.
I would prefer to put the responsibility of calling `parseFloat` in the calling function and not changing the type at all here.